### PR TITLE
Add DSRV information that supports IBC relayer on Persistence

### PR DESCRIPTION
**Pull request by gnongs**
Add DSRV information that supports IBC relayer on Persistence

Signed-off-by: Jinhong Choi <jinhongdev@gmail.com>

### CODE REVIEW

**Comment by dneorej** [comment]
@gnongs can you please also include DSRV as an operator on the relevant channel(s) [here](https://github.com/persistenceOne/persistence-docs/blob/main/build/relayers/ibc-channels.md) please?

**Comment by gnongs** [comment]
> 



> @gnongs can you please also include DSRV as an operator on the relevant channel(s) [here](https://github.com/persistenceOne/persistence-docs/blob/main/build/relayers/ibc-channels.md) please?

@dneorej Updated!

**Comment by dneorej** [comment]
Thanks, can you also resolve the conflict please?

**Review by dneorej — CHANGES_REQUESTED** [review]
Thanks @gnongs, can you also resolve the conflict please?

**Comment by gnongs** [comment]
> Thanks @gnongs, can you also resolve the conflict please?

@dneorej Resolved!

**Comment by dneorej** [comment]
> > Thanks @gnongs, can you also resolve the conflict please?
> 
> @dneorej Resolved!

Seems like you're overwriting the changes done by Cosmonaut Stakes (adding his info + improving the formatting of the icons). Can you reverse that and just add your changes on top of his? Thanks

**Comment by gnongs** [comment]
> > > Thanks @gnongs, can you also resolve the conflict please?
> > 
> > 
> > @dneorej Resolved!
> 
> @gnongs Seems like you're overwriting the changes done by Cosmonaut Stakes (adding his info + improving the formatting of the icons). Can you reverse that and just add your changes on top of his? Thanks

@dneorej Updated!

File: build/relayers/ibc-relayers.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ We encourage everyone to follow and support our IBC-relayer operators
...
+| [DSRV](http://dsrvlabs.com/) | <p><a href="mailto:validator@dsrvlabs.com">validator@dsrvlabs.com</a><br><img src="https://user-images.githubusercontent.com/95667791/163506878-e7e278b2-1cf6-4dac-bc13-e92f63600ee0.png" alt="Twitter" data-size="line"> Twitter <a href="http://twitter.com/dsrvlabs">@dsrvlabs</a><br><img src="https://user-images.githubusercontent.com/95667791/163506639-7d27e947-94c7-4740-bb46-8e5791e9fc6c.png" alt="Telegram" data-size="line"> Telegram @kim201212</p> | <p>🔒 Delegate to DSRV Validators on: <a href="https://wallet.keplr.app/#/osmosis/stake?modal=detail&#x26;chainId=osmosis-1&#x26;validator=osmovaloper1wlagucxdxvsmvj6330864x8q3vxz4x025rraa6">Osmosis</a>, Cosmos Hub, Presistence, Axelar, Agoric, Injective, Shetu, Crescent, Provenance, Tgrade, Terra, Stargaze, Chihuahua, Umee, Assetmantle |
```
Comment (dneorej) [inline build/relayers/ibc-relayers.md:19]: We're almost there @gnongs - if you can just include the same formatting settings for the icons as the other rows in the table, then we're good to go (now they look very big). Thanks!

**Review by dneorej — CHANGES_REQUESTED** [review]

File: build/relayers/ibc-relayers.md (hunk 1 above)
Comment (gnongs) [reply build/relayers/ibc-relayers.md]: @dneorej I changed it !

**Review by dneorej — APPROVED** [review]

**Pull request merged**